src/wasm.h

```diff
@@ -1427,15 +1427,15 @@ class Function {
   }
 };
 
+enum class Kind {
```

Comment:
I guess the design repo doesn't care about specifying anything other than the import and export types (i.e. Binaryen is different because it wants to model them in a more general IR). I really do think `Global` is the right description for the kind of thing we are trying to model; if I were doing my own naming I'd call overall thing `Global` or something else with global in the name (since it's global to the instance), and then have `GlobalVariable`. It's just unfortunate that the spec/design already has `Global`. Maybe `ModuleValue` or something with `Module` in the name?